roles/swift_exporter/files/swift_exporter.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python3
+import swiftclient
+import yaml
+import logging
+import sys
+from prometheus_client import start_http_server, Gauge
+from collections import defaultdict
+import time
+
+#Define prometheus_metrics
+gauge_container_count = Gauge('swift_container_count','Number of container in an account', ['account_tags'])
+gauge_object_count = Gauge('swift_object_count', 'Number of object in an account', ['account_tags'])
+gauge_used_bytes = Gauge('swift_used_bytes', 'Used bytes in an account', ['account_tags'])
+gauge_quota_bytes = Gauge('swift_quota_bytes', 'quota bytes in an account', ['account_tags'])
+gauge_container_connect = Gauge('swift_container_connect', 'Connection to container', ['account_tags'])
+
+## Read yaml config
+def get_config(configfile):
+    with open(configfile, 'r') as stream:
+        try:
+            config_obj = yaml.safe_load(stream)
+            return config_obj
+        except yaml.YAMLError as exc:
+            logging.critical("Error reading yaml config file %s", exc)
+            sys.exit(1)
+        except OSError as exc:
+            logging.critical(" yaml config file Not found %s", exc)
+            sys.exit(1)
+
+## Connect with swift object storage
+def swift_connect(user,key,authurl):
+    try:
+        conn = swiftclient.Connection(
+            user=user,
+            key=key,
+            authurl=authurl,
+        )
+        resp_headers, containers = conn.get_account()
+    except Exception as exc:
+        logging.critical(" Exception during connecting to object storage %s", exc)
+        resp_headers=None
+        containers= None
+    return resp_headers,containers
+
+## Get container size        
+def calculate_container_size(resp_headers,containers):
```

Comment:
consider renaming to `get_account_stats`. This could also return a `namedtuple` which would make the rest of the code much simpler as you wouldn't need all the dict access by key